### ISSUE TITLE
Intallation spelling error in .storybook/welcome.stories.js 

### ISSUE THREAD

**Issue by MalikMAlna** [react-materialize/react-materialize#1182]
### Description

Spelling error, should be "Installation" not "Intallation" in .storybook/welcome.stories.js on line 61

### Steps to Reproduce

N/A

**Expected behavior:** [What you expect to happen]

Installation should be spelt right on the page.

**Actual behavior:** [What actually happens]

Installation is spelt wrong on the page for the subheading of the page.

### Versions

**react-materialize:** current version - v3.9.7
**materialize-css:** current version
**react:**  current version

I can fix this right now for you all if you want. I've already got the fork set up and the spelling correction fixed in a commit on my end. 

Basically, I'm waiting on the O.K. to create the PR.

**Comment by MalikMAlna** [comment]
Nevermind, passed in the PR to master. 

Let me know if that's not okay.
